ccd-definition/AuthorisationCaseEvent.json

```diff
@@ -152,5 +152,12 @@
     "CaseEventID": "uploadStandardDirections",
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "CaseEventID": "sendToGatekeeper",
```

Comment:
Are users from other roles supported to see that action in case history?

Reply:
Spoke to Sam Tate regarding this, and she said that it should just be the HMCTS admin that can see the event history of sending to gatekeeper